### PR TITLE
Bulk doc update of Speech to text API v3.0

### PR DESCRIPTION
**Pull request by bexxx**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [ ] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [X] Other, please clarify: Documentation and example updates
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  Code is already deployed, this is just a doc update.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  ASAP
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [X] No, no need to refresh for updates in this PR

### Contribution checklist:
- [X] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [X] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [X] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  -  [ ] Ensure to copy the existing version into new directory structure for first commit (including refactoring) and then push new changes including version updates in separate commits. This is required to review the changes efficiently.
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by bexxx** [comment]
@jhendrixMSFT the "new" response code was always returned and missed to be documented when the initial v3.0 swagger version was published.

So currently we could break customers  by sending unexpected response codes and we try fixing this by documenting the one.
This is used mainly during our rollouts to see whether we can swap the deployments to a new version, I could not monitor external requests against this route.

File: specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json:6158]: Is there a reason this doesn't have the `x-ms-error-response` annotation?  Seems weird to treat a 503 as a success.

Comment (bexxx) [reply specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json:6158]: The response type is the same as the 200/OK response and not an error type. We did not add this intentionally because of this.

Comment (mkistler) [reply specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json:6158]: This is very strange.  I have never seen a `503` response that returns the same response body as a `200-OK`.  Are you sure that is correct?  If the service is "temporarily unavailable" (the meaning of the 503 status code), what created a request that _looks just like_ a response from the service?

**Review by mkistler — CHANGES_REQUESTED** [review]
I really believe the `503` response is incorrect and should be changed, but I'm willing to listen to a good explanation of why that's not the case.

File: specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json
Comment (mikekistler) [reply specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json:6158]: @bexxx Can you please explain how a 503 response can be a successful response?  That is highly unusual!

Comment (Fraegle) [reply specification/cognitiveservices/data-plane/Speech/SpeechToText/stable/v3.0/speechtotext.json:6158]: The route in question is the health route. The One API response body knows three states: Healthy, Degraded and Unhealthy. If a service instance is degraded or unhealthy, we take it from the load balancer and if it doesn't become healthy, the instance gets restarted. For this we use the [health check feature of Azure App Services](https://docs.microsoft.com/en-us/azure/app-service/monitor-instances-health-check). This feature requires to return a 5xx in case the service is degraded or unhealthy. 

The challenge is, that the documented response type for the health check is different from the standard error response type. In this case the response body contains the health response type. This is the same response type, that also gets returned in case the service is healthy and we return 200.

**Review by mikekistler — APPROVED** [review]
Looks good. 👍

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
I just noticed these changes are in a stable swagger.  @johanste is this acceptable?  Has the service been deployed yet?

**Comment by Fraegle** [comment]
> I just noticed these changes are in a stable swagger. @johanste is this acceptable? Has the service been deployed yet?

The service is already deployed with this behavior. We just missed to document it in Swagger.

**Comment by bexxx** [comment]
@jhendrixMSFT how can we close on this?
The function here has been approved after the explanation that this is the health route. We also have a breaking change approval from the azure review board. What else is blocking the merge of this PR? Thanks.

**Pull request merged**
